core/modules/image/config/image.settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "_config_name": "image.settings",
+    "svg_sanitizer_class": "enshrined\\svgSanitize\\Sanitizer"
```

Comment:
What was the intention behind that extra config file? Is it supposed to contain other items?
To me it seems a bit overdone to create an extra json file for one config option.

Oh, and don't forget the update hook for existing sites, as otherwise there won't be any svg sanitizing. And admins wouldn't even know.

Reply:
> What was the intention behind that extra config file? Is it supposed to contain other items?

I feel it would be helpful to expose some way to either change or extend the sanitizer. This is the easiest way, I suspect. Alternatives considered:
- just this: a config file entry in Image module
- a config entry in another module (File, System, etc). But the sanitizer is just used by Image fields so if Image is disabled, its not needed. Still could go in System.
- just force this sanitizer, dont allow extending or changing, so no config file needed
- create a new hook `hook_svg_sanitizer()`
- add a configuration form to select available sanitizers provided by contrib or custom (would still save to config)

For something that is unlikely to be frequently changed, I thought just the config entry should be good enough. Any other ideas welcome.

Reply:
Right, no image module - no sanitizer... :thinking:

I don't think, we need an admin UI for sanitizer class selection. I'm not even sure, if we need the option to replace the class, anyway. But if so, and given it's a setting hardly ever changed, ... a hook instead of config? That seems worth considering.

If someone wants to _replace_ the class, there's some coding required, anyway.

Reply:
Thanks @indigoxela I'm leaving this open for other opinions for now.

Reply:
I agree about simplifying the sanitizer. No need for a UI or config. And perhaps don't even need the class. We might only need to pass it through something like: `$svg_contents = filter_xss($svg_contents, array('svg', 'use', 'title', 'desc', 'defs', 'linearGradient', 'stop', 'rect', 'circle', 'path'));`, similar to how it's done in this PR https://github.com/backdrop/backdrop/blob/df47d525efd5f074b4a73161e20c80612cd83909/core/includes/icon.inc. Maybe there could be a hook so that a contrib module could override that.

Reply:
I think the weight of opinion is firmly against this sanitizer, so, its gone! 😄